samples/apps/copilot-chat-app/importdocument/Program.cs

```diff
@@ -26,30 +26,29 @@ public static void Main(string[] args)
             return;
         }
 
-        var fileOption = new Option<FileInfo>(name: "--file", description: "The file to import to document memory store.")
+        var filesOption = new Option<List<FileInfo>>(name: "--files", description: "The files to import to document memory store.")
```

Comment:
Consider IEnumerable here 

```suggestion
        var filesOption = new Option<IEnumerable<FileInfo>>(name: "--files", description: "The files to import to document memory store.")
```